README.md

```diff
@@ -121,22 +121,11 @@ this larger namespace.
 
    Item
 
-   (optional | single)
-
-   Links to an external file containing a transcript.  The mime type of the file should be given - such as text/plain, text/html, etc.
-
-   All attributes are required.
-
-
-- **\<podcast:captions language="[language code]" type="text/srt">**[url to a SRT captions file]**\</podcast:captions>**
-
-   Item
-
-   (optional | single)
+   (optional | multiple)
 
-   Links to an industry standard closed-caption/subtitle file format.
+   Links to an external file containing a transcript.  The mime type of the file should be given - such as `text/plain`, `text/html`, `application/srt`, `application/json`.
 
-   All attributes are required.
+   The language of the transcript can be overwritten by providing the optional language attribute. All other attributes are required.
```

Comment:
What about making the language optional? In most cases we have 3 transcript types available for each episode in the RSS feed. This would reduce the amount of code required as long as we are comfortable defaulting to the same language as the podcast.

Reply:
That's fair.  Let's do that.  I may clarify your statement here as we go refine the wording of things.

Reply:
Sure thing! I'm happy to help, but am more comfortable with code than english 🤣